### PR TITLE
Add 3 9 test runner

### PR DESCRIPTION
**Pull request by ssantana-ns**
Add a python 3.9 runner

### CODE REVIEW

**Review by ns-bdesimone — APPROVED** [review]
LGTM - Do we need to build for all versions or just these two?

**Review by jdrake — APPROVED** [review]

File: .circleci/workflows/pull-request-tests.yml [hunk 1]
```diff
@@ -5,3 +5,6 @@ jobs:
         branches:
           ignore:
             - main
+      matrix:
+        parameters:
+          python_version: ["3_6", "3_9"]
```
Comment (ns-bdesimone) [inline .circleci/workflows/pull-request-tests.yml:10]: 🔥

**Review by ns-bdesimone — DISMISSED** [review]
LGTM

**Review by ns-bdesimone — APPROVED** [review]
LGTM

**Pull request merged**